src/jbake/content/officers-and-board.html

```diff
@@ -9,42 +9,87 @@
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="Bob Paulin is an independent consultant and speaker that has been developing open source software for the past 10 years. Bob has presented at large international conferences such as ApacheCon, JavaOne and Devnexus. He frequently shares his knowledge and opinions on the Java Pub House and Jave Off Heap podcasts. Bob is actively involved in the Chicago Java User Group (CJUG) and CJUG4Kids presenter/organizer. Bob is a passionate member of the ASF and donates his time to Apache Felix and Apache Tika as a committer and PMC respectively. When not coding, Bob enjoys coaching football, robotics, and spending time with his wife and 4 kids.">Bob Paulin</a>
                     </div>
                 </div>
             </div>
 
             <div class="card text-white bg-primary mb-3" style="max-width: 15rem;">
                 <div class="card-body">
                     <h5 class="card-title" style="text-align: center">Board Member</h5>
+                    <img src="images/cjug_small.jpeg" onload="this.onload=null; this.src=gravatar('paul.nauman@cjug.org', '150');" style="display: block;margin-left: auto;margin-right:auto"/>
+                    <br/>
+                    <div style="text-align:center">
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="">Paul Nauman</a>
+                    </div>
+                </div>
+            </div>
+
+            <div class="card text-white bg-primary mb-3" style="max-width: 15rem;">
+                <div class="card-body">
+                    <h5 class="card-title" style="text-align: center">President (2019)</h5>

```

Comment:
Should say emeritus. President (2019) is unclear IMO

Reply:
Thanks Bob for the approval ! For now, i am leaving it as-is. And we will revisit it and the whole page in next set of updates..